internal/brokers/localbroker.go

```diff
@@ -5,40 +5,60 @@ import (
 	"errors"
 )
 
-//nolint:unused // We still need localBroker to implement the brokerer interface, even though this type will not be interacted with by the daemon.
+// even though this type will not be interacted with by the daemon.
```

Comment:
Did you use the auto-correction from the linter?

Reply:
Oh... I had tried it yes, I thought it didn't do anything... But apparently it has changed something :(

Reply:
Yeah, I completely understand your pain... It happened to me once also.

Reply:
So... Even changing this manually the auto-formatter set it to the same way. So I need to just keep `nolint::unused` as separated line here.